alf/nest/nest.py

```diff
@@ -797,6 +797,67 @@ def _traverse_transform(nested, levels):
         nested=nested, levels=field.split('.') if field else [])
 
 
+def transform_nests(nests, field, func):
+    """Transform the node of each of the nest in nests indicated by ``field``
+        using ``func``.
+
+    This function can be used to transform multiple nests, and perform
+        transformations with inter-nest interactions.
+
+        .. code-block:: python
+
+            res1, res2 = transform_nests([nest1, nest2], 'a.b',
+                                lambda x: (x[0] * x[1], x[0] + x[1]))
```

Comment:
My understanding is that this is a drop in replacement to

```python
map(lambda x: x[0] * x[1], x[0] + x[1], [obj.a['b'] for obj in [nest1, nest2]])
```

If so, my humble opinion is that using native python code above here is simpler and more readable.

Reply:
> My understanding is that this is a drop in replacement to
> 
> ```python
> map(lambda x: x[0] * x[1], x[0] + x[1], [obj.a['b'] for obj in [nest1, nest2]])
> ```
> 
> If so, my humble opinion is that using native python code above here is simpler and more readable.

Thanks for the comment. If I understand correctly, the native python code does not produce a result that preserves the original nest structure?.

One of the reason for using ``transform_xxx`` functions is to transform the specified field of the nest and keep others and the whole nest structure unchanged.

Reply:
Ah sorry I did not notice that there is already `transform_nest` as an analogy. I understand now that without `_replace` a namedtuple cannot be altered.